src/engine/resources.ts

```diff
@@ -17,12 +30,37 @@ export interface BanishSource extends CombatResource {
   do: Item | Skill;
 }
 
+const availableClub: Item =
+  Item.all().find(
+    (i) =>
+      have(i) &&
+      canEquip(i) &&
+      weaponHands(i) === 2 &&
+      (itemType(i) === "club" || (have($skill`Iron Palm Technique`) && itemType(i) === "sword"))
+  ) ?? $item`amok putter`;
+
 export const banishSources: BanishSource[] = [
   {
     name: "System Sweep",
     available: () => have($skill`System Sweep`),
     do: $skill`System Sweep`,
   },
+  {
+    name: "Batter Up!",
+    available: () => have($skill`Batter Up!`) && myFury() >= 5,
+    prepare: (): void => {
+      if (
+        have($skill`Iron Palm Technique`) &&
+        !have($effect`Iron Palms`) &&
+        itemType(availableClub) === "sword"
+      ) {
+        useSkill($skill`Iron Palm Technique`);
+      }
+      retrieveItem(availableClub);
+    },
+    equip: availableClub,
```

Comment:
You need to make sure that this gets put in the mainhand

Reply:
Thanks!